content-src/components/DiscoveryStreamComponents/List/List.jsx

```diff
@@ -60,8 +60,16 @@ export function _List(props) {
 
   const recs = feed.data.recommendations;
 
-  let recMarkup = recs.slice(props.recStartingPoint, props.items).map((rec, index) => (
-    <ListItem {...rec} key={`ds-list-item-${index}`} index={index} type={props.type} dispatch={props.dispatch} />)
+  let recMarkup = recs.slice(0, props.items).map((rec, index) => (
```

Comment:
I guess we all missed the slice change to 0 instead of `recStartingPoint` 😊 Fixed in #4740